### ISSUE TITLE
launch_xctest : not running ui tests

### ISSUE THREAD

**Issue by sergd2005** [facebook/FBSimulatorControl#405]
Hi everyone, I am new to fbsimctl - maybe it is simple issue - however I am trying to launch my ui tests using following command (fyi I read that post already https://github.com/facebook/FBSimulatorControl/issues/336 - I am able to launch wda with fbsimctl wihout any issues) :

fbsimctl 2480467B-6F4A-4984-A94B-17CE409D39A7 launch_xctest /Users/Shared/DerivedData/TestRunner-fukaqtyppayflrfaecyablvvfmjl/Build/Products/Debug-iphonesimulator/TestAppUITests-Runner.app/PlugIns/TestAppUITests.xctest sdsoft.TestApp

Short version of output :

Test Plan Started
Started initilizing for UI testing.
Test Plan Ended

In few words no tests are executed at all. However I have single test there that taps button. In overall I have impression that this command is only suitable for launching wda. Please correct me if I am wrong. Thx. 

Here is full log file:
[log.txt](https://github.com/facebook/FBSimulatorControl/files/982007/log.txt)

**Comment by chandu97309** [comment]
I am also having the same issue. In the simulator logs i see this:

*** Assertion failure in -[XCUIApplication init], /Library/Caches/com.apple.xbs/Sources/XCTest_Sim/XCTest-12124/Sources/UITesting/XCUIApplication.m:64

This is happening when XCUIApplication().launch() is executed.

**Comment by sergd2005** [comment]
I ll attach logs from sim also

**Comment by vmaxim** [comment]
```facebook/fb/fbsimctl HEAD-3848920```

Command:
```fbsimctl 1F9E9BE0-0ACE-487D-8E69-A458A6DFCD8D launch_xctest  /Users/admin/Library/Developer/Xcode/DerivedData/WebDriverAgent-brdadhpuduowllgivnnvuygpwhzy/Build/Products/WebDriverAgentRunner_iphonesimulator11.0-x86_64.xctestrun as.df.rt --port 8200 -- listen```

Output:
```'["1F9E9BE0-0ACE-487D-8E69-A458A6DFCD8D", "launch_xctest", "/Users/admin/Library/Developer/Xcode/DerivedData/WebDriverAgent-brdadhpuduowllgivnnvuygpwhzy/Build/Products/WebDriverAgentRunner_iphonesimulator11.0-x86_64.xctestrun", "as.df.rt", "--port", "8200", "--", "listen"]' does not match '[fbsimctl print [action]```
